app/build.gradle

```diff
@@ -30,6 +30,7 @@ android {
     packagingOptions {
         exclude 'META-INF/rxjava.properties'
     }
+    buildToolsVersion '26.0.2'
```

Comment:
travis fails because of this version of build tools.
Maybe we can change it or you can update Travis box/agent @ramonharrison

Reply:
Yes, I've reopened your original PR #280 -- the Travis request seems to be working now. 

I opened #283 to update build tools in the Travis script. As soon as that's merged we should be good to retest your PR.